cmd/booster-bitswap/run.go

```diff
@@ -83,14 +83,25 @@ var runCmd = &cli.Command{
 		remoteStore := remoteblockstore.NewRemoteBlockstore(bapi)
 		// Create the server API
 		port := cctx.Int("port")
-		server := NewBitswapServer(port, remoteStore)
-
+		repoDir := cctx.String(FlagRepo.Name)
+		host, err := setupHost(ctx, repoDir, port)
+		if err != nil {
+			return fmt.Errorf("setting up libp2p host: %w", err)
```

Comment:
If the peer key is not found, I think this will output an error like "setting up libp2p host: libp2p.key not found"
I'd suggest we show something more informative, eg
```
if os.IsNotExist(err) {
	return nil, fmt.Errorf("booster-bitswap has not been initialized. Run the booster-bitswap init command.")
}
```